COBOL Programming Course #2 - Advanced Topics/COBOL Programming Course #2 - Advanced Topics.md

```diff
@@ -225,6 +225,115 @@ COMP-2 supersedes COMP-1 for more precise scientific data storage as well as com
 
 **Note** : [This](https://www.ibm.com/support/pages/how-display-hexadecimal-using-cobol) COBOL program can display the hexadecimal contents (and hence the exact internal representation) of a field. You can declare binary, packed decimal or zoned variable (or anything else, for that matter), do arithmetic with them and use the program to see how they are internally stored. 
 
+\newpage
+# Dynamic-Length Item
+So far during this course, we have only explored data items that have a fixed length. In other words, you need to define the length you need for each data type. But in this section, we will explore a fairly new concept in Enterprise COBOL - dynamic-length items.
+
+Enterprise COBOL v6.3 supports dynamic-length items, which are items whose logical length might change at runtime.
```

Comment:
As you mentioned in the PR, much of this content is from the Enterprise COBOL for z/OS documentation - https://www.ibm.com/docs/en/cobol-zos/6.3?topic=relationships-dynamic-length-items. Let's credit this source up-front. Additionally, much of this content is identical to that source. We should either quote the source directly or construct this chapter in our own words from what we learned from the source (and still credit it).

Reply:
Sure. Would something like [the one on Course 1](https://github.com/openmainframeproject/cobol-programming-course/blob/master/COBOL%20Programming%20Course%20%231%20-%20Getting%20Started/COBOL%20Programming%20Course%20%231%20-%20Getting%20Started.md#professional-manuals) suffice? Otherwise, I can add them to Course 1 too.

Reply:
I like the idea of including something like the Professional Manuals section of Course 1 in Course 2. I still believe that when we adapt content, we should use our own words or directly quote the source.

Reply:
Agreed. Will re-write the paragraphs on this PR sometimes this week. Additionally, I will also put the Professional Manuals section.

Reply:
Thank you!!

Reply:
Added to c81c78a213ebafd7a4abc4596d354d545f13b51e. When Course 2 is finalized, might need to shift things around to tidy the placing and formatting of the Professional Manuals section.